tests/src/test/scala/system/basic/WskSdkTests.scala

```diff
@@ -82,6 +84,76 @@ class WskSdkTests extends TestHelpers with WskTestHelpers {
     }
   }
 
+  it should "download docker sdk when blackbox.tar.gz exists, and docker name should be quoted in error." in {
```

Comment:
Could use `Seq` to squash this test and `"download ios sdk when OpenWhiskIOSStarterApp.zip exists, and ios name should be quoted in error.` into one test.